src/Sylius/Component/User/Context/CustomerContextInterface.php

```diff
@@ -19,7 +19,7 @@
 interface CustomerContextInterface
 {
     /**
-     * @return CustomerInterface|null
+     * @return null|CustomerInterface
```

Comment:
There is no EOT in Sylius PR discussions. :) I do not think we should remove this interface.